pkg/logs/message/message.go

```diff
@@ -74,3 +76,17 @@ func (m *Message) GetStatus() string {
 func (m *Message) GetLatency() int64 {
 	return time.Now().UnixNano() - m.IngestionTimestamp
 }
+
+// GetHostname returns the hostname to applied the given log message
+func (m *Message) GetHostname() string {
+	if m.Lambda == nil {
+		hostname, err := util.GetHostname(context.TODO())
```

Comment:
It looks like the body of this conditional is just `processor.getHostname`, to which this diff removes all references.  So, I'd suggest:
 * remove that utility function
 * refactor this to treat the `Lambda` case as the special case, and the no-lambda case as the default (i.e., `if m.Lambda != nil ..`).  Then when google invents hostless computing, we can add `if m.Hostless != nil { return "hosts-are-so-2021" }` 😄)

Reply:
Thanks for the review!
✅  for treating lambda as the special case
This methods is called there for instance : https://github.com/DataDog/datadog-agent/pull/8519/files#diff-40507f3cd5a5844dd542647018eebd84a22d6fa6fb19736ae758b643edb2ede4R44 so I can't remove it

Reply:
I was referring to removing the (lowercase, non-object-receiver) `processor.getHostname` function.  Sorry, there were a lot of functions referred to here, all with the same name :)

Reply:
By the way, nice job on the refactor and test coverage!

Reply:
In this case those methods have already been removed in this PR here.
Maybe you didn't see it because the two files are removed?
<img width="699" alt="Screen Shot 2021-08-04 at 2 47 18 PM" src="https://user-images.githubusercontent.com/864493/128237861-9d4364c7-9cad-452c-883d-2786c8f8f437.png">

Reply:
ahah no worries I just wanted to make sure that we're talking about the same thing :)